From 50e51833acfdddbb024649de0a7d80ae0f192974 Mon Sep 17 00:00:00 2001
From: Michel Thebeau <michel.thebeau@windriver.com>
Date: Fri, 26 Apr 2013 11:26:26 -0400
Subject: [PATCH 37/39] vxlan: refactor netlink_skb_parms.portid to pid

An upstream commit refactors netlink_skb_parms.pid to portid.  This
hasn't happened yet in our kernel.  pid is what we want.

Signed-off-by: Michel Thebeau <michel.thebeau@windriver.com>
---
 drivers/net/vxlan.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/vxlan.c b/drivers/net/vxlan.c
index c5ed2a6..7c82160 100644
--- a/drivers/net/vxlan.c
+++ b/drivers/net/vxlan.c
@@ -428,7 +428,7 @@ static int vxlan_fdb_dump(struct sk_buff *skb, struct netlink_callback *cb,
 				goto skip;
 
 			err = vxlan_fdb_info(skb, vxlan, f,
-					     NETLINK_CB(cb->skb).portid,
+					     NETLINK_CB(cb->skb).pid,
 					     cb->nlh->nlmsg_seq,
 					     RTM_NEWNEIGH,
 					     NLM_F_MULTI);
-- 
1.7.9.7

